x-pack/plugins/maps/server/saved_objects/maps_telemetry.ts

```diff
@@ -37,8 +37,8 @@ export const mapsTelemetrySavedObjects: SavedObjectsType = {
               avg: { type: 'long' },
             },
           },
-          layerTypesCount: { dynamic: 'true', properties: {} },
-          emsVectorLayersCount: { dynamic: 'true', properties: {} },
+          layerTypesCount: { type: 'object', enabled: false },
```

Comment:
This will prevent any queries on fields inside `layerTypesCount`. If we do need to be able to query this data we will have to change the data that's being indexed so that it conforms to a fixed schema instead of dynamically adding new fields.